web-gateway/app/controllers/ItemForm.java

```diff
@@ -31,30 +31,21 @@
         List<ValidationError> errors = new ArrayList<>();
         
         try {
-            Currency c = Currency.valueOf(currency);
-
+        	Currency c = Currency.valueOf(currency);
```

Comment:
Please make sure you're using only space characters to indent, not tabs. You can probably set up your editor or IDE to do this automatically... let us know what tool you're using if you'd like a pointer on how to set it up.